packages/serverless-components/nextjs-cdk-construct/__tests__/construct.test.ts

```diff
@@ -0,0 +1,169 @@
+import "@aws-cdk/assert/jest";
+import { SynthUtils } from "@aws-cdk/assert";
+import { Stack } from "@aws-cdk/core";
+import path from "path";
+import { NextJSLambdaEdge } from "../src";
+import { Runtime } from "@aws-cdk/aws-lambda";
+import { Certificate } from "@aws-cdk/aws-certificatemanager";
+import { HostedZone } from "@aws-cdk/aws-route53";
+
+describe("CDK Construct", () => {
```

Comment:
New to CDK myself, but these LGTM in terms of style

Reply:
Im sure there are many many more test cases, however, keeping light for now whilst this is still "experimental" and subject to change.